services-camel/src/main/resources/config.properties

```diff
@@ -17,6 +17,8 @@ cdr.enhancement.jp2.mimetype=image/jp2
 cdr.enhancement.postIndexingDelay=5000
 cdr.enhancement.processingThreads=2
 
+cdr.replication.locations=
```

Comment:
Did we ever decide whether we need this variable and, if so, what it's values should be?